docs/design/eth1.md

```diff
@@ -0,0 +1,381 @@
+# [ETH1 - Information Retrival] Design
```

Comment:
@nkryuchkov lets maybe have it in the eth package?

Reply:
@lior-blox I agree. Moved